src/Feature.js

```diff
@@ -0,0 +1,32 @@
+function Feature(props) {
+  const featureHash = feature + '-' + idx;
+  const options = this.props.features[feature].map(item => {
+    const itemHash = slugify(JSON.stringify(item));
+    return (
+      <div key={itemHash} className="feature__item">
+        <input
+          type="radio"
+          id={itemHash}
+          className="feature__option"
+          name={slugify(feature)}
+          checked={item.name === this.state.selected[feature].name}
+          onChange={e => this.updateFeature(feature, item)}
+        />
+        <label htmlFor={itemHash} className="feature__label">
+          {item.name} ({USCurrencyFormat.format(item.cost)})
+        </label>
+      </div>
+    );
+  });
+
+  return (
+    <fieldset className="feature" key={featureHash}>
+      <legend className="feature__name">
+        <h3>{feature}</h3>
+      </legend>
+      {options}
+    </fieldset>
+  );
+};
```

Comment:
I left the code exactly as it was before (just pasted), it won't work yet because we don't have any of those variables.